Addition of release documentation (#1396)
vitessio/website: content/en/docs/releases/_index.md

---
title: Releases

> description: Learn about how Vitess releases

Reviewer: ```suggestion
description: Learn about how the Vitess maintainers manage releases
```

Reviewer: Other option:
```
Learn about how Vitess releases work
```

Author: Fixed via 29f34634

weight: 3000
---

> The maintainers team of Vitess supports the last 3 major releases of Vitess.

Reviewer: ```suggestion
The Vitess maintainers support the last 3 major releases of Vitess.
```

> Each major release is maintained by the maintainers during 1 year.

Reviewer: ```suggestion
Each major release is maintained for 1 year.
```

> > The release lifecycle of Vitess evolved over the years, each iteration is written down in a VEP (Vitess Enhancement Proposal).

Reviewer: ```suggestion
> The release lifecycle of Vitess has evolved over the years, each iteration is written down in a VEP (Vitess Enhancement Proposal).
```

>
> VEPs [#1](https://github.com/vitessio/enhancements/blob/main/veps/vep-1.md), [#2](https://github.com/vitessio/enhancements/blob/main/veps/vep-2.md), and [#5](https://github.com/vitessio/enhancements/blob/main/veps/vep-5.md) all mention the release lifecycle of Vitess.
> 
> The last up-to-date VEP is [VEP #5](https://github.com/vitessio/enhancements/blob/main/veps/vep-5.md).

## Release History

### v16.0
- **Current version:** v16.0.0
- **Release date:** 2023-02-28
- **End of life:** 2024-02-28
- **Patch releases:** v16.0.0 

[//]: # (TODO: add a link to the v16.0.0 release above)

### v15.0
- **Current version:** v15.0.2
- **Release date:** 2022-12-14
- **End of life:** 2023-10-25

> - **Patch releases:** [v15.0.0](https://github.com/vitessio/vitess/releases/tag/v15.0.0), [v15.0.1](https://github.com/vitessio/vitess/releases/tag/v15.0.1), [v15.0.2](https://github.com/vitessio/vitess/releases/tag/v15.0.2)

Reviewer: 15.0.0 is technically not a patch release, so how do you want to reorganize this?
Maybe add another line?
```
**GA Release** v15.0.0 Release date <blah>
```

Reviewer: The same thing applies to v14 and v13.

Author: Agreed, I am thinking of reorganizing it this way:
```md
- **Current version:** [v15.0.2](https://github.com/vitessio/vitess/releases/tag/v15.0.2) (2022-12-14)
- **GA release:** [v15.0.0](https://github.com/vitessio/vitess/releases/tag/v15.0.0) (2022-10-25)
- **End of life:** 2023-10-25
- **Patch releases:** [v15.0.1](https://github.com/vitessio/vitess/releases/tag/v15.0.1), [v15.0.2](https://github.com/vitessio/vitess/releases/tag/v15.0.2)
```

Author: Fixed via 29f34634
